tests/console/sshd.pm

```diff
@@ -144,9 +155,17 @@ sub run {
     # Restore the /etc/ssh/sshd_config
     assert_script_run 'cp /etc/ssh/sshd_config{_before,}';
 
+    # Kill $ssh_testman to stop all SSH sessions
+    script_run "killall -u $ssh_testman";
```

Comment:
Please keep the original `assert_script_run`. An unguarded `script_run` call can eventually fail, produce red borders which don't fail tests. Even if we want to explicity ignore the exit code I consider `assert_script_run` the much better default unless you want to explicitly check the exit code of `script_run`.

Reply:
OK, I didn't know that.

Reply:
Like this `assert_script_run "killall -u $ssh_testman || true";` could fail :rofl: But yes `assert_script_run` should be used